src/scripts.js

```diff
@@ -16,164 +14,61 @@ const userForms = document.querySelector('.todays-metrics');
-      postSleepSubmission(dataToPost);
-    }
-  }
-
-  function createSleepObject(dateInput, sleepAmount, sleepQuality) {
-    const formattedDate = dateInput.value.replace(/-/g,"/");
-    return {"userID": user.id, "date": formattedDate, "hoursSlept": parseInt(sleepAmount.value), "sleepQuality": parseInt(sleepQuality.value)}
-  }
-
-  function postSleepSubmission(dataToPost) {
-    fetch("https://fe-apps.herokuapp.com/api/v1/fitlit/1908/sleep/sleepData", {
-      method: 'POST',
-      headers: {
-  	'Content-Type': 'application/json'
-    },
-      body: JSON.stringify(dataToPost),
-    })
-    .then(response => response.json())
-    .then(message => handlePostSuccess('.sleep'))
-    .catch(error => console.log(error.message))
-  }
-
-  function evaluateActivityInput() {
-    const dateInput = document.querySelector('.activity-date-input')
-    const stepCount = document.querySelector('.steps-walked-input')
-    const minutesActive = document.querySelector('.minutes-active-input')
-    const stairCount = document.querySelector('.stairs-input')
-    if (dateInput.value !== "" && minutesActive.value !== "" && stepCount.value !== "" && stairCount.value !== "") {
-      const dataToPost = createActivityObject(dateInput, stepCount, minutesActive, stairCount)
-      postActvitySubmission(dataToPost);
-    }
-  }
-
-  function createActivityObject(dateInput, stepCount, minutesActive, stairCount) {
-    const formattedDate = dateInput.value.replace(/-/g,"/");
-    return {"userID": user.id, "date": formattedDate, "numSteps": parseInt(stepCount.value), "minutesActive": parseInt(minutesActive.value), "flightsOfStairs": parseInt(stairCount.value)}
-  }
-
-  function postActvitySubmission(dataToPost) {
-    console.log(dataToPost);
-    fetch("https://fe-apps.herokuapp.com/api/v1/fitlit/1908/activity/activityData", {
-      method: 'POST',
-      headers: {
-  	'Content-Type': 'application/json'
-    },
-      body: JSON.stringify(dataToPost),
-    })
-    .then(response => response.json())
-    .then(message => handlePostSuccess('.activity'))
-    .catch(error => console.log(error.message))
-  }
-
-function startApp(userRepo, hydrationRepo, sleepRepo, activityRepo, userNowId, userNow, today, randomHistory, historicalWeek) {
-  console.log(userNowId)
-  historicalWeek.forEach(instance => instance.insertAdjacentHTML('afterBegin', `Week of ${randomHistory}`));
-  addInfoToSidebar(userNow, userRepo);
-  getHyrdationElements(userNowId, hydrationRepo, today, userRepo, randomHistory)
-  getSleepElements(userNowId, sleepRepo, today, userRepo, randomHistory);
-  let winnerNow = makeWinnerID(activityRepo, userNow, today, userRepo);
-  getFriendInfoElements(userNowId, activityRepo, userRepo, today, randomHistory, userNow);
-  getActivityElements(userNowId, activityRepo, today, userRepo, randomHistory, userNow, winnerNow);
-}
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I really like your use of a physical barrier between different sections of code.